src/runtime-rs/crates/hypervisor/src/dragonball/inner.rs

```diff
@@ -143,8 +148,11 @@ impl DragonballInner {
 
         // set boot source
         let kernel_path = self.config.boot_info.kernel.clone();
+        let initrd_path = self.config.boot_info.initrd.clone();
```

Comment:
Nit: maybe using `&self.config.boot_info.initrd` is better than `self.config.boot_info.initrd.clone()`, but the L150 is indeed using `.clone()`.